steel/docs/create-steel-app/README.md

```diff
@@ -0,0 +1,50 @@
+# create-steel-app
+
+## What does this script do?
+
+The [create-steel-app] script will allow you to set up the [erc20-counter] example locally in one command. This example will act as your skeleton project structure for your own application.  Once the script is finished, you can run through a test workflow with either local proving or Bonsai proving. You can also learn more about Steel by reading the [documentation]. 
+
+## Dependencies
+
+Make sure to have the following installed:
+
+1. [Rust]
+2. [Foundry]
+3. [cargo-risczero]
+
+## Usage
+
+`sh -c "$(curl -fsSL https://raw.githubusercontent.com/risc0/risc0-ethereum/refs/heads/main/steel/docs/create-steel-app/create-steel-app)"`
+
+The script will automatically detect your current `cargo-risczero` version and use that for the corresponding version of the `erc20-counter` example. You also have the manual choice between between two release versions: [1.0] and [1.1].
+
+Once the script is finished running, you should:
+
+```
+cd PROJECT_NAME
+cargo build
+forge build
+``` 
+
+After this, you can export the necessary Bonsai environment variables:
```

Comment:
```suggestion
After this, you can export the necessary Bonsai environment variables if you'd like to use Bonsai. Otherwise, local proving will be used:
```